### PR TITLE
feat(wallet): Send Error Improvements with Tooltips

### PR DESCRIPTION
**Pull request by Douglashdaniel**
## Description 
Send Error Improvements with Tooltips

1) Added a `Tooltip` to the send button that will only appear during 2 disabled states.
- Input amount is `0` tooltip will display `Amount must be greater than 0`
- Address is `empty` tooltip will display `To address is required`
2) Changed checksum `warning` message to `This address cannot be verified. Proceed?`.
3) Changed checksum `error` message to `Address did not pass verification. Please try again, replacing lowercase letters with uppercase.`
4) Both checksum `warning` and `error` messages now have a `Learn more.` button that will link to the same `Support Page`

Resolves <https://github.com/brave/brave-browser/issues/21053>

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

https://user-images.githubusercontent.com/40611140/154601204-bc28665a-127e-4926-ba26-ee1b150bc2c5.mov

### CODE REVIEW

File: components/brave_wallet_ui/components/buy-send-swap/send/index.tsx [hunk 1]
```diff
@@ -83,6 +84,17 @@ function Send (props: Props) {
     return amountWei.gt(selectedAssetBalance)
   }, [selectedAssetBalance, selectedAssetAmount, selectedAsset])
 
+  const tooltipMessage = React.useMemo((): string => {
+    if (parseFloat(selectedAssetAmount) === 0 ||
```
Comment (onyb) [inline components/brave_wallet_ui/components/buy-send-swap/send/index.tsx:88]: Let's use the new `Amount` tools:

```ts
const amountWrapped = new Amount(selectedAssetAmount)
if (amountWrapped.isUndefined() || amountWrapped.isZero()) {
  // ...
}
```

File: components/brave_wallet_ui/components/buy-send-swap/swap-input-component/index.tsx [hunk 2]
```diff
@@ -230,6 +232,10 @@ function SwapInputComponent (props: Props) {
       .format(6, true)
     : ''
 
+  const onClickLearnMore = () => {
+    window.open('https://support.brave.com/hc/en-us/articles/4441999049101', '_blank')
```
Comment (onyb) [inline components/brave_wallet_ui/components/buy-send-swap/swap-input-component/index.tsx:236]: Does this work inside the panel?

**Review by onyb — CHANGES_REQUESTED** [review]

File: components/brave_wallet_ui/components/buy-send-swap/send/index.tsx (hunk 1 above)
Comment (Douglashdaniel) [reply components/brave_wallet_ui/components/buy-send-swap/send/index.tsx]: Updated!

File: components/brave_wallet_ui/components/buy-send-swap/swap-input-component/index.tsx (hunk 2 above)
Comment (Douglashdaniel) [reply components/brave_wallet_ui/components/buy-send-swap/swap-input-component/index.tsx]: Good catch! Updated to use Chrome API.

**Review by muliswilliam — APPROVED** [review]

**Review by onyb — APPROVED** [review]
Discussed the wording changes internally. Good to merge as-is. If the strings aren't getting too long, you can try these too:

```
Address did not pass verification (invalid checksum). Please try again, replacing lowercase letters with uppercase.

Address cannot be verified (missing checksum). Proceed?
```

**Pull request merged**